doc/source/tutorial/tutorial_presenting_larray_objects.ipyml

```diff
@@ -106,6 +119,24 @@ cells:
     teens
 
 
+- markdown: |
+    <div class="alert alert-warning">
+    
+    **Warning:** Mixing slices and individual labels inside the `[ ]` will generate a **tuple of groups** instead of unique group.<br>If you want to create a unique group using both slices and individual labels, you need to use the `.union()` method (see below).   
```

Comment:
s/a tuple of/several distinct/ ?
s/of unique/of a single/
s/a unique/a single/